action/protocol/staking/handler_stake_migrate.go

```diff
@@ -25,10 +25,13 @@ import (
 	"github.com/iotexproject/iotex-core/state"
 )
 
-func (p *Protocol) handleStakeMigrate(ctx context.Context, act *action.MigrateStake, csm CandidateStateManager) ([]*action.Log, []*action.TransactionLog, uint64, uint64, error) {
-	actLogs := make([]*action.Log, 0)
-	transferLogs := make([]*action.TransactionLog, 0)
-	insGas, err := act.IntrinsicGas()
+func (p *Protocol) handleStakeMigrate(ctx context.Context, elp action.Envelope, csm CandidateStateManager) ([]*action.Log, []*action.TransactionLog, uint64, uint64, error) {
```

Comment:
`handleStakeMigrate(ctx, common, act, csm)`
As the envelope has been opened, passing the `common` and `act` as separate parameters is more reasonable